app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -117,7 +126,53 @@ fun OverviewScreen(
       },
       modifier = Modifier.testTag("overviewScreen"),
       topBar = {
-        TopAppBar(title = { Text(text = "Your trips") }, modifier = Modifier.testTag("topBarTitle"))
+        TopAppBar(
+            title = {
+              Box(
+                  modifier = Modifier.fillMaxWidth().fillMaxHeight(),
+                  contentAlignment = Alignment.CenterStart) {
+                    if (!searchVisible) {
+                      Text(
+                          text = "Your trips",
```

Comment:
Could you please use the string resources folder here instead of hardcoded strings? :)